recipes/eodatasets3/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "eodatasets3" %}
+{% set version = "0.29.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/opendatacube/eo-datasets/raw/conda-release/dist/eodatasets3-0.29.5.tar.gz
```

So that the bot can send you automatic version updates.
```suggestion
  url: https://github.com/opendatacube/eo-datasets/raw/conda-release/dist/eodatasets3-{{ version }}.tar.gz
```